flake.nix

```diff
@@ -41,10 +41,10 @@
     let
       inherit (self) outputs;
       forAllSystems = nixpkgs.lib.genAttrs [
-        "aarch64-darwin"
+        "x86_64-linux"
         "aarch64-linux"
         "x86_64-darwin"
-        "x86_64-linux"
+        "aarch64-darwin"
```

Comment:
I think this change should probably be reverted to avoid unnecessary merge conflict errors on other branches.